openupgrade_scripts/scripts/hr_recruitment/15.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.is_module_installed("hr_recruitment_notification"):
```

Comment:
This is not correct, as in this moment, `hr_recruitment_notification` module doesn't exist (it was merged).

Just do the renaming. If not originally installed, the SQL operation won't update anything.